### DISCUSSION TITLE
Grafana Mimir Ruler error:

### DISCUSSION THREAD

**Discussion by felipefrx** [grafana/mimir#8983] [Help and support]
I'm trying to configure my ruler in Grafana Mimir, but it return this error:

![Screenshot from 2024-08-12 17-46-50](https://github.com/user-attachments/assets/c6c63b27-ecf7-42a9-b3c6-158e3e3fba55)

**Comment by charleskorn** [comment]
How is your data source configured in Grafana? I suspect it is missing the `X-Scope-OrgID` header.

**Reply by felipefrx** [reply]
I configured it. The metrics is showing.
